--- a/SeriousProton-EE-2022.10.28/src/io/http/websocket.h
+++ b/SeriousProton-EE-2022.10.28/src/io/http/websocket.h
@@ -4,10 +4,7 @@
 #include <nonCopyable.h>
 #include <io/network/tcpSocket.h>
 #include <unordered_map>
-
-#ifndef EMSCRIPTEN
 #include <memory>
-#endif
 
 namespace sp {
 namespace io {

--- a/SeriousProton-EE-2022.10.28/src/io/network/address.h
+++ b/SeriousProton-EE-2022.10.28/src/io/network/address.h
@@ -3,7 +3,7 @@
 
 #include <stringImproved.h>
 #include <list>
-
+#include <cstdint>
 
 namespace sp {
 namespace io {

