--- EmptyEpsilon-EE-2021.03.16/src/spaceObjects/asteroid.h	2021-03-16 14:36:34.000000000 +0100
+++ EmptyEpsilon-EE-2021.03.16/src/spaceObjects/asteroid.h_pacthed	2021-03-28 08:38:40.332730717 +0200
@@ -16,6 +16,7 @@ public:
     virtual void draw3D() override;
 
     virtual void drawOnRadar(sf::RenderTarget& window, sf::Vector2f position, float scale, float rotation, bool long_range) override;
+    virtual bool canHideInNebula()  override { return false; }
 
     virtual void collide(Collisionable* target, float force) override;
 
