--- EmptyEpsilon-EE-2021.03.16/src/screens/crew6/scienceScreen.cpp	2021-03-16 14:36:34.000000000 +0100
+++ EmptyEpsilon-EE-2021.03.16/src/screens/crew6/scienceScreen.cpp_patched	2021-03-27 23:33:41.278919582 +0100
@@ -44,7 +44,7 @@ ScienceScreen::ScienceScreen(GuiContaine
     // Draw the science radar.
     science_radar = new GuiRadarView(radar_view, "SCIENCE_RADAR", my_spaceship ? my_spaceship->getLongRangeRadarRange() : 30000.0, &targets);
     science_radar->setPosition(-270, 0, ACenterRight)->setSize(GuiElement::GuiSizeMax, GuiElement::GuiSizeMax);
-    science_radar->setRangeIndicatorStepSize(5000.0)->longRange()->enableWaypoints()->enableCallsigns()->enableHeadingIndicators()->setStyle(GuiRadarView::Circular)->setFogOfWarStyle(GuiRadarView::NebulaFogOfWar);
+    science_radar->setRangeIndicatorStepSize(3000.0)->longRange()->enableWaypoints()->enableCallsigns()->enableHeadingIndicators()->setStyle(GuiRadarView::Circular)->setFogOfWarStyle(GuiRadarView::NebulaFogOfWar);
     science_radar->setCallbacks(
         [this](sf::Vector2f position) {
             if (!my_spaceship || my_spaceship->scanning_delay > 0.0)
