From 69d93e6acdae3259755924f9d35e7e5ae949d377 Mon Sep 17 00:00:00 2001
From: Daid <daid303@gmail.com>
Date: Fri, 25 Jun 2021 11:15:48 +0200
Subject: [PATCH] GLM library is moved to seriousproton

---
 CMakeLists.txt | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f65642f..119a94b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -105,19 +105,6 @@ if(WITH_DISCORD)
   target_link_libraries(discord INTERFACE discord_h)
 endif()
 
-# GLM
-set(GLM_BASE_PATH "${EXTERNALS_DIR}")
-
-if(NOT EXISTS "${GLM_BASE_PATH}/glm/CMakeLists.txt")
-  set(GLM_ZIP "${DOWNLOADS_DIR}/glm.zip")
-  file(DOWNLOAD "https://github.com/g-truc/glm/releases/download/0.9.9.8/glm-0.9.9.8.zip" "${GLM_ZIP}" TIMEOUT 60 TLS_VERIFY ON)
-
-  file(MAKE_DIRECTORY "${GLM_BASE_PATH}/glm")
-  execute_process(COMMAND ${CMAKE_COMMAND} -E tar -xf "${GLM_ZIP}" WORKING_DIRECTORY "${GLM_BASE_PATH}")
-endif()
-
-add_subdirectory("${GLM_BASE_PATH}/glm" "${PROJECT_BINARY_DIR}/glm" EXCLUDE_FROM_ALL)
-
 set(SOURCES
     src/main.cpp
     src/threatLevelEstimate.cpp
@@ -347,7 +334,7 @@ target_link_libraries(opengl
 
 target_link_libraries(${PROJECT_NAME}
     PUBLIC
-        seriousproton glm::glm opengl
+        seriousproton opengl
         "$<$<BOOL:${ANDROID}>:log;android>"
         "$<$<BOOL:${WIN32}>:iphlpapi>"
         "$<$<BOOL:${WITH_DISCORD}>:discord_h>"
--
libgit2 1.0.1

