From 872ef2667c0e4a3afab1dfe7e977b8dbd539aebd Mon Sep 17 00:00:00 2001
From: Michal Schorm <mschorm@redhat.com>
Date: Sun, 27 Jun 2021 23:45:45 +0200
Subject: [PATCH] Remove a forgotten left over from the move of CMake MinGW Crash Logger code to SeriousProton project

---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 119a94b..63770de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,8 +61,6 @@ endif()
 set(SERIOUSPROTON_WITH_JSON ON CACHE BOOL "" FORCE)
 add_subdirectory("${SERIOUS_PROTON_DIR}" "${PROJECT_BINARY_DIR}/SeriousProton" EXCLUDE_FROM_ALL)
 
-message(STATUS "Crash Logger is: ${ENABLE_CRASH_LOGGER}")
-
 set(EXTERNALS_DIR "${PROJECT_BINARY_DIR}/externals")
 set(DOWNLOADS_DIR "${PROJECT_BINARY_DIR}/downloads")
 file(MAKE_DIRECTORY "${EXTERNAL_DIR}" "${DOWNLOADS_DIR}")
--
libgit2 1.0.1

