From ee0cd42bfe5fd20b8339e8e02eb7f69766168d57 Mon Sep 17 00:00:00 2001
From: Michal Schorm <mschorm@redhat.com>
Date: Mon, 28 Jun 2021 00:44:33 +0200
Subject: [PATCH] CMake: Explicitly set OpenGL version to LEGACY to deal with the CMake policy CMP0072

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 63770de..1465001 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,8 +51,9 @@ message(STATUS "EmptyEpsilon Version = ${PROJECT_VERSION}")
 
 # Dependencies
 
-# Setup OpenGl
+# Setup OpenGL
 if(NOT ANDROID)
+    set(OpenGL_GL_PREFERENCE LEGACY)
     find_package(OpenGL REQUIRED)
 endif()
 
--
libgit2 1.0.1

