--- EmptyEpsilon-EE-2021.03.16/src/screenComponents/scanningDialog.cpp	2021-03-16 14:36:34.000000000 +0100
+++ EmptyEpsilon-EE-2021.03.16/src/screenComponents/scanningDialog.cpp_patched	2021-03-21 05:37:39.083814178 +0100
@@ -51,7 +51,7 @@ void GuiScanningDialog::onDraw(sf::Rende
         {
             if (!box->isVisible())
             {
-                box->show();
+//                box->show();
                 scan_depth = 0;
                 setupParameters();
             }
