--- EmptyEpsilon-EE-2021.03.16/src/screens/crew6/relayScreen.cpp	2021-03-16 14:36:34.000000000 +0100
+++ EmptyEpsilon-EE-2021.03.16/src/screens/crew6/relayScreen.cpp_patched	2021-03-28 07:34:04.797412212 +0200
@@ -25,7 +25,7 @@ RelayScreen::RelayScreen(GuiContainer* o
 {
     targets.setAllowWaypointSelection();
     radar = new GuiRadarView(this, "RELAY_RADAR", 50000.0f, &targets);
-    radar->longRange()->enableWaypoints()->enableCallsigns()->setStyle(GuiRadarView::Rectangular)->setFogOfWarStyle(GuiRadarView::FriendlysShortRangeFogOfWar);
+    radar->longRange()->enableWaypoints()->enableCallsigns()->setStyle(GuiRadarView::Rectangular)->setFogOfWarStyle(GuiRadarView::NebulaFogOfWar);
     radar->setAutoCentering(false);
     radar->setPosition(0, 0, ATopLeft)->setSize(GuiElement::GuiSizeMax, GuiElement::GuiSizeMax);
     radar->setCallbacks(
